tests/driver_sx1280/main.c

```diff
@@ -0,0 +1,368 @@
+/*
+ * Copyright (C) 2022 Inria
+ * Copyright (C) 2020-2022 Université Grenoble Alpes
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     tests
+ * @{
+ *
+ * @file
+ * @brief       Test Application For SX1280 Driver
+ *
+ * @author      Francisco Molina
+ * @author      Nicolas Albarel
+ * @author      Didier Donsez
+ * @author      Olivier Alphand
+ * @author      Aymeric Brochier
```

Comment:
I think you can leave yourself as the sole author here no?, from the original PR you completely re-did the example, in any case please add an email for the remaining authors.